docs/pages/build/eas-json.md

```diff
@@ -212,6 +227,7 @@ The schema of a build profile for a generic iOS project looks like this:
 - `fastlane` - version of fastlane
 - `cocoapods` - version of CocoaPods
 - `env` - environment variables that should be set during the build process (should only be used for values that you would commit to your git repository, i.e: not passwords or secrets).
+- `cache` configures paths that will be saved and restored in the next build, the cache can be explicitly invalidated by updating the value of the `key` field. Values in `customPaths` support both absolute and relative paths, where relative paths are resolved from the directory with eas.json. If you set `cacheDefaultPaths` to true or leave `cache` config unspecified Podfile.lock will be cached by default. This feature is intended for caching values that require a lot of computation, caching something like `node_modules` will increase the build time significantly.
```

Comment:
```suggestion
- `cache` configures paths that will be saved and restored in the next build. The cache can be explicitly invalidated by updating the value of the `key` field. Values in `customPaths` support both absolute and relative paths, where relative paths are resolved from the directory with `eas.json`. If you set `cacheDefaultPaths` to true, or leave the `cache` config unspecified, `Podfile.lock` will be cached by default. This feature is intended for caching values that require a lot of computation. For example, caching `node_modules` will decrease the build time significantly.
```

Reply:
if you override `cache` key but don't provide a `cacheDefaultPaths` key, does it still default to `true`? if not, i think we should do that in order to prevent people from making a mistake when they only want to bump the cache key

Reply:
yes, it's still defaulting to true